esbuild.config.mjs

```diff
@@ -41,6 +39,12 @@ const context = await esbuild.context({
   treeShaking: true,
   outfile: "main.js",
   plugins: [svgPlugin(), wasmPlugin],
+  define: {
+    global: "window",
+  },
+  alias: {
+    events: "./null-events.js",
```

Comment:
I needed to add this in order to make `PouchDB` work correctly. Maybe we can switch out to another DB that doesn't rely on `events` internally and clean this up.